sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/GeneratedFileFilter.java

```diff
@@ -32,13 +32,23 @@
   private static final Logger LOG = Loggers.get(GeneratedFileFilter.class);
 
   private final AbstractGlobalProtobufFileProcessor globalReportProcessor;
+  private final boolean analyzeGeneratedCode;
 
-  public GeneratedFileFilter(AbstractGlobalProtobufFileProcessor globalReportProcessor) {
+  public GeneratedFileFilter(AbstractGlobalProtobufFileProcessor globalReportProcessor, AbstractSolutionConfiguration configuration) {

```

Comment:
I'm confused. The new `analyzeGeneratedCode` settings has been added to both the existing `AbstractConfiguration` and the new `AbstractSolutionConfiguration`, but this filter only applies to the new class. Why is that?

Reply:
I agree, to make things clearer I would keep only the getter in `AbstractSolutionConfiguration`

Reply:
sorry, I already did that in commit _Add AbstractSolutionConfiguration UT (e81d89f507116f75a41abf58bfbba665582271ac)_